indico/modules/events/registration/controllers/management/reglists_test.py

```diff
@@ -0,0 +1,102 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2023 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+import pytest
+from flask import request
+from werkzeug.exceptions import UnprocessableEntity
+
+from indico.modules.events.registration.controllers.management.fields import _fill_form_field_with_data
+from indico.modules.events.registration.controllers.management.reglists import RHRegistrationCreate, RHRegistrationEdit
+from indico.modules.events.registration.models.form_fields import RegistrationFormField
+from indico.modules.events.registration.models.items import RegistrationFormSection
+from indico.modules.events.registration.util import create_registration
+from indico.util.date_time import now_utc
+
+
+pytest_plugins = 'indico.modules.events.registration.testing.fixtures'
+
+
+def test_registration_edit_override_required(
+    db, dummy_regform, dummy_user, app_context, create_user
```

Comment:
```suggestion
    db, dummy_regform, app_context, create_user
```

This fixture is unused